tls/s2n_connection.h

```diff
@@ -281,6 +281,10 @@ struct s2n_connection {
 
     /* application protocols overridden */
     struct s2n_blob application_protocols_overridden;
+
+    /* Cookie extension data */
+    uint8_t cookie_data[S2N_MAX_COOKIE_SIZE];
+    uint16_t cookie_data_len;
```

Comment:
This will prevent us from working with random size cookies !

Reply:
Same as as above, using a stuffer to hold variable sized data.